src/coverlet.collector/InProcDataCollection/CoverletInProcDataCollector.cs

```diff
@@ -36,17 +38,14 @@ public void TestSessionEnd(TestSessionEndArgs testSessionEndArgs)
 
                 try
                 {
+                    _eqtTrace.Verbose($"Calling ModuleTrackerTemplate.UnloadModule for '{injectedInstrumentationClass.Assembly.FullName}'");
                     var unloadModule = injectedInstrumentationClass.GetMethod(nameof(ModuleTrackerTemplate.UnloadModule), new[] { typeof(object), typeof(EventArgs) });
                     unloadModule.Invoke(null, new[] { null, EventArgs.Empty });
+                    _eqtTrace.Verbose($"Called ModuleTrackerTemplate.UnloadModule for '{injectedInstrumentationClass.Assembly.FullName}'");
                 }
                 catch (Exception ex)
                 {
-                    // Throw any exception if unload fails
-                    if (EqtTrace.IsErrorEnabled)
-                    {
-                        EqtTrace.Error("{0}: Failed to unload module with error: {1}", CoverletConstants.InProcDataCollectorName, ex);
-                    }
-
+                    _eqtTrace.Error("{0}: Failed to unload module with error: {1}", CoverletConstants.InProcDataCollectorName, ex);
```

Comment:
@vagisha-nidhi I removed check `EqtTrace.IsErrorEnabled` here and on other place because I don't understand how I can define switch level using command `dotnet test --collect:"Xplat code coverage" --diag:"diag.txt"`
Again they're not used on other place(out-of-proc collector), are used only for Info...
Can you explain how it works?Or How I can pass log level for `EqtTrace` to `dotnet test`

Reply:
@vagisha-nidhi I merged...let me know if it's a problem.